### ISSUE TITLE
Name table seems broken in Windows 10

### ISSUE THREAD

**Issue by mcpeanutbutter** [kencrocken/FiraCodeiScript#7]
Thank you for this cool font! Unfortunately I cannot get it to work properly with Windows 10 and IntelliJ. 

Have a look at the following comment:
https://github.com/source-foundry/Hack/issues/50#issuecomment-137058006

I experience a similar problem with the FiraCodeiScript font in Windows 10. Windows recognizes the regular and the bold version and places them in the same family. However, the italic font is _not_ recognized as part of the family. It does not appear in the Windows font folder until I delete the other two family members (which are properly grouped together). 

While some programs (e.g. Adobe Illustrator) properly utilizes the italic version of the font anyway, unfortunately, IntelliJ IDEA (and likely other JetBrain IDEs) do not. 

This issue can probably be fixed by adjusting the name table and the font names.

**Comment by timnevits** [comment]
Yes -- I'm seeing a similar issue -- only on my Windows 10 machine I see Bold and Italic grouped together with Regular missing.  At the moment, the font does not work with Intellij on Windows 10 sadly.

**Comment by zimmah** [comment]
I don't know if this has been fixed yet, but it appears not, because on windows the italics look like the same font as the rest.

Sad, because I loved this font on my mac.
